### PR TITLE
Fix visual changes

### PR DESCRIPTION
**Pull request by Vardan-Alaverdyan**

### CODE REVIEW

File: src/view/Attestation.jsx [hunk 1]
```diff
@@ -61,7 +61,7 @@ class Attestation extends Component {
                             <th>Txid</th>
                             <th>Merkle Root</th>
                             <th>Confirmed</th>
-                            <th>Time</th>
+                            <th>Time UTC</th>
```
Comment (nkostoulas) [inline src/view/Attestation.jsx]: (UTC)

File: src/view/LatestAttestation.jsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ class LatestAttestation extends Component {
                           <tr>
                               <th><span className="lh1rem mr-auto">Txid</span></th>
                               <th><span className="lh1rem mr-auto">Merkle Root</span></th>
-                              <th className="lastHead"><span className="lh1rem mr-auto ">Time</span></th>
+                              <th className="lastHead"><span className="lh1rem mr-auto ">Time UTC</span></th>
```
Comment (nkostoulas) [inline src/view/LatestAttestation.jsx]: (UTC)

File: src/view/LatestAttestationInfo.jsx [hunk 3]
```diff
@@ -45,16 +45,35 @@ class LatestAttestationInfo extends Component {
...
+                    </div>
+                    <div className="overview-item">
+                        <div>
+                            <p>Average Fee (USD)</p>
+                            <strong>0.34 USD</strong>
+                        </div>
+                    </div>
+
+                    <div className="overview-item overview-item--time">
+                        <p>Time</p>
```
Comment (nkostoulas) [inline src/view/LatestAttestationInfo.jsx]: Time (UTC)

File: src/view/PriceCBT.jsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import Sockette from 'sockette';
+import {Link} from 'react-router-dom';
+
+class PriceCBT extends Component {
```
Comment (nkostoulas) [inline src/view/PriceCBT.jsx]: I thought we renamed all "CBT" to "BTC".

Comment (nkostoulas) [reply src/view/PriceCBT.jsx]: Also since we are no longer displaying the "BTC" price the WS calls should be disabled.

Comment (Vardan-Alaverdyan) [reply src/view/PriceCBT.jsx]: I've replaced all CBT with BTC before. The reason that we have CBT here again, is that this branch was created before my changes. Anyway, I've changed everything again to BTC.

Regarding WS calls: We are not displaying BTC price, but we displaying the Average Fee (USD) which is calculated based on fee and BTC price

File: src/view/Attestation.jsx (hunk 1 above)
Comment (Vardan-Alaverdyan) [reply src/view/Attestation.jsx]: Done

File: src/view/LatestAttestation.jsx (hunk 2 above)
Comment (Vardan-Alaverdyan) [reply src/view/LatestAttestation.jsx]: Done

File: src/view/LatestAttestationInfo.jsx (hunk 3 above)
Comment (Vardan-Alaverdyan) [reply src/view/LatestAttestationInfo.jsx]: Done

File: src/view/PriceCBT.jsx (hunk 4 above)
Comment (nkostoulas) [reply src/view/PriceCBT.jsx]: Ah right sorry. Thanks!

**Pull request merged**